packages/eddsa-poseidon/rollup.iife.config.ts

```diff
@@ -35,13 +33,5 @@ export default {
         }
     ],
     external: [],
-    plugins: [
-        typescript({ tsconfig: "./build.tsconfig.json" }),
-        commonjs(),
-        nodeResolve({
-            preferBuiltins: true
-        }),
-        nodePolyfills({ include: null }),
-        cleanup({ comments: "jsdoc" })
-    ]
+    plugins: [typescript({ tsconfig: "./build.tsconfig.json" }), commonjs(), cleanup({ comments: "jsdoc" })]
```

Comment:
`@zk-kit/eddsa-poseidon` includes `blake-hash` in its bundle, which uses the NodeJS Buffer class. If we want to remove these Rollup plugins, we should copy the `blake-hash` code we need and replace Buffer with the npm `buffer` package. Just a possible proposal.